mbedtls.mod/mbedtls.bmx

```diff
@@ -566,6 +585,20 @@ Type TX509Cert
 		Return bmx_mbedtls_x509_crt_parse(certPtr, buf, buflen)
 	End Method
 
+	Rem
+	bbdoc: Parses a file with one or more certificates (usually .pem extension)
+	End Rem
+	Method ParseFile:Int(path:String)
```

Comment:
Rather than directly implementing the "file" api, it might be better to have a TStream interface that ParseFile(String) could call, and which would end up calling the xxx_parse() functions with the contents of a byte array... that gives you more options as to the location of the file.

Reply:
Something like that? cdee293